Codeinwp/neve
# Site title font size

### Summary
Added a new control for the Logo component, **Site Title Font Size**. The label looks like this so we avoid any new string additions.
Please note that this control is not dependent on others, it's not hiding based on the Show title toggles because the toggle have the live refresh option enabled and in this case, the active callback property will just get the first value of iy and not update if the toggle value changes. 

### Will affect the visual aspect of the product
NO

### Screenshots 
https://vertis.d.pr/i/JRbKSG

### Test instructions
- Please make sure the default font size for the title doesn't change when you update to this version
- Go to Customizer and change the font size for the title and try other units too.

## Check before Pull Request is ready:



Closes #3914.
